### ISSUE TITLE
Cannot load library as ESM module

### ISSUE THREAD

**Issue by eddieantonio** [digitallinguistics/transliterate#49]
Hello! I cannot figure out how to load this library as recommended, using the experimental ESM module support. Can it be made available using the old-school commonjs (`module.exports`/`require()`) style?

Here's a minimum reproduction of my issue. I'm using macOS Catalina:

```
 $ node --version
v14.6.0
 $ npm --version
6.14.6
 $ rm -rf node_modules
 $ grep "module" ./package.json
  "type": "module",
 $ npm install @digitallinguistics/transliterate
npm WARN Plains-Cree@0.1.0 No repository field.
npm WARN Plains-Cree@0.1.0 No license field.

+ @digitallinguistics/transliterate@0.2.1
added 1 package from 1 contributor and audited 1 package in 0.332s
found 0 vulnerabilities

 $ echo "import { transliterate } from '@digitallinguistics/transliterate';" > index.js
 $ node index.js
internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of URL. Received undefined
    at fileURLToPath (internal/url.js:1347:11)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:833:21)
    at Loader.resolve (internal/modules/esm/loader.js:82:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:226:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:51:40)
    at link (internal/modules/esm/module_job.js:50:36) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by dwhieb** [comment]
I think #46 should fix this as well as make it available using CJS modules.

Duplicate of #46

**Comment by dwhieb** [comment]
Just fixed this in the v0.2.2 release - should work as an ES module now (but not a CJS module - [esm](https://www.npmjs.com/package/esm) is probably your easiest bet if you need it as a CJS module).
